### PR TITLE
For for the moving average scores

### PR DESCRIPTION
**Pull request by Eugene-hu**
- Fixes the scatter in the validator to reward only those that queried 
- Adds in additional normalization for followup and answer rewards

### CODE REVIEW

File: openvalidators/forward.py [hunk 1]
```diff
@@ -341,12 +341,12 @@ async def forward(self):
...
     scattered_followup_rewards = (
-        torch.zeros((self.metagraph.n), dtype=torch.float32).to(self.device).scatter(0, followup_uids, followup_rewards)
+        self.moving_averaged_scores.scatter(0, followup_uids, followup_rewards)
     )
     scattered_answer_rewards = (
-        torch.zeros((self.metagraph.n), dtype=torch.float32).to(self.device).scatter(0, answer_uids, answer_rewards)
+        self.moving_averaged_scores.scatter(0, answer_uids, answer_rewards)
     )
-    rewards = scattered_followup_rewards + scattered_answer_rewards
+    rewards = (scattered_followup_rewards + scattered_answer_rewards)/2
```
Comment (adriansmares) [inline openvalidators/forward.py:349]: So you're dividing by 2 since the UIDs outside of `followup_uids` and `answer_uids` will be added twice, but you're also halving the score for the UIDs in `followup_uids` and `answer_uids`. 

Why not scatter the second round of rewards over the first one ? `followup_uids` and `answer_uids` are disjoint, and then you count the scores of the UIDs which were not queried only once, without changing other scores.

**Review by opentaco — APPROVED** [review]

**Review by shibshib — APPROVED** [review]

**Review by isabella618033 — APPROVED** [review]
LGTM!
